src/Components/Counter/__tests__/Counter.test.js

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { Counter } from '../Counter';
+
+describe('Counter tests', () => {
```

Comment:
It's clear, that these are tests, no need to have "tests" in suite's title:

```suggestion
describe('Counter', () => {
```